web-api-db-ctrl/src/elastic.rs

```diff
@@ -48,21 +48,35 @@ pub(crate) async fn setup_legacy_tenant(
     default_index: &str,
     tenant_id: &TenantId,
 ) -> Result<(), Error> {
-    elastic
-        .with_index("_aliases")
-        .request(Method::POST, [], [])
-        .json(&json!({
-          "actions": [
-            {
-              "add": {
-                "index": default_index,
-                "alias": tenant_id,
-              }
-            }
-          ]
-        }))
+    let response = elastic
+        .with_index(default_index)
+        .request(Method::GET, [], [])
```

Comment:
❓ should we use a `Method::HEAD` request instead to have an even more light-weight "ping" response?
```suggestion
        .request(Method::HEAD, [], [])
```

Reply:
during some manual testing this didn't work correctly once for me as elastic did hung on the response for whatever reason :see_no_evil: 

also this is done once during startup, the size difference of the response really doesn't matter